[pcl] build failure

### Operating system

windows 11

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
`./vcpkg install pcl[cuda,surface-on-nurbs,tools,visualization,vtk]`
```

### Failure logs

Installing 1/2 pcl[core,cuda,surface-on-nurbs,tools,visualization,vtk]:x64-windows@1.15.0...
Building pcl[core,cuda,surface-on-nurbs,tools,visualization,vtk]:x64-windows@1.15.0...
-- Using cached PointCloudLibrary-pcl-pcl-1.15.0.tar.gz
-- Cleaning sources at C:/Users/user/opt/vcpkg/buildtrees/pcl/src/pcl-1.15.0-327f82be19.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/Users/user/opt/vcpkg/downloads/PointCloudLibrary-pcl-pcl-1.15.0.tar.gz
-- Applying patch add-gcc-version-check.patch
-- Applying patch fix-check-sse.patch
-- Applying patch fix-numeric-literals-flag.patch
-- Applying patch install-layout.patch
-- Applying patch install-examples.patch
-- Applying patch fix-clang-cl.patch
-- Using source at C:/Users/user/opt/vcpkg/buildtrees/pcl/src/pcl-1.15.0-327f82be19.clean
-- Found external ninja('1.12.1').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/Users/user/scoop/apps/cmake/3.31.6/bin/cmake.exe --build . --config Release --target install -- -v -j13
    Working Directory: C:/Users/user/opt/vcpkg/buildtrees/pcl/x64-windows-rel
    See logs for more information:
      C:\Users\user\opt\vcpkg\buildtrees\pcl\install-x64-windows-rel-out.log

### Additional context

in file C:\Users\user\opt\vcpkg\buildtrees\pcl\install-x64-windows-rel-out.log, the error is:
C:\Users\user\opt\vcpkg\buildtrees\pcl\src\pcl-1.15.0-327f82be19.clean\visualization\include\pcl/visualization/impl/registration_visualizer.hpp(183): error C2039: 'chrono_literals': is not a member of 'std'
  and in file registration_visualizer.hpp chrono_literals was only loaded but not used, if it is useable please add #include<chrono> to this file.

--- a/ports/pcl/add-chrono-includes.patch
+++ b/ports/pcl/add-chrono-includes.patch
@@ -1,0 +1,117 @@
+diff --git a/apps/src/ppf_object_recognition.cpp b/apps/src/ppf_object_recognition.cpp
+index b12ac2a0f..9889d0c6a 100644
+--- a/apps/src/ppf_object_recognition.cpp
++++ b/apps/src/ppf_object_recognition.cpp
+@@ -9,6 +9,7 @@
+ #include <pcl/segmentation/sac_segmentation.h>
+ #include <pcl/visualization/pcl_visualizer.h>
+ 
++#include <chrono>
+ #include <thread>
+ 
+ using namespace pcl;
+diff --git a/examples/stereo/example_stereo_baseline.cpp b/examples/stereo/example_stereo_baseline.cpp
+index bf21c6f8f..594ab2465 100644
+--- a/examples/stereo/example_stereo_baseline.cpp
++++ b/examples/stereo/example_stereo_baseline.cpp
+@@ -1,3 +1,4 @@
++#include <chrono>
+ #include <thread>
+ 
+ #include <pcl/stereo/stereo_matching.h>
+diff --git a/tools/obj_rec_ransac_accepted_hypotheses.cpp b/tools/obj_rec_ransac_accepted_hypotheses.cpp
+index f75bd66a7..f884cd9c9 100644
+--- a/tools/obj_rec_ransac_accepted_hypotheses.cpp
++++ b/tools/obj_rec_ransac_accepted_hypotheses.cpp
+@@ -58,6 +58,7 @@
+ #include <vtkHedgeHog.h>
+ #include <vtkMatrix4x4.h>
+ #include <algorithm>
++#include <chrono>
+ #include <cstdio>
+ #include <thread>
+ #include <vector>
+diff --git a/tools/obj_rec_ransac_hash_table.cpp b/tools/obj_rec_ransac_hash_table.cpp
+index bde3d9413..88b959944 100644
+--- a/tools/obj_rec_ransac_hash_table.cpp
++++ b/tools/obj_rec_ransac_hash_table.cpp
+@@ -53,6 +53,7 @@
+ #include <vtkDataArray.h>
+ #include <vtkPointData.h>
+ #include <vtkGlyph3D.h>
++#include <chrono>
+ #include <cstdio>
+ #include <thread>
+ 
+diff --git a/tools/obj_rec_ransac_model_opps.cpp b/tools/obj_rec_ransac_model_opps.cpp
+index d82cbfb16..e6b27a67d 100644
+--- a/tools/obj_rec_ransac_model_opps.cpp
++++ b/tools/obj_rec_ransac_model_opps.cpp
+@@ -52,6 +52,7 @@
+ #include <vtkDataArray.h>
+ #include <vtkPointData.h>
+ #include <vtkHedgeHog.h>
++#include <chrono>
+ #include <cstdio>
+ #include <thread>
+ 
+diff --git a/tools/obj_rec_ransac_orr_octree.cpp b/tools/obj_rec_ransac_orr_octree.cpp
+index c21a193c4..6f55c0916 100644
+--- a/tools/obj_rec_ransac_orr_octree.cpp
++++ b/tools/obj_rec_ransac_orr_octree.cpp
+@@ -63,6 +63,7 @@
+ #include <vtkRenderWindow.h>
+ #include <vector>
+ #include <list>
++#include <chrono>
+ #include <cstdlib>
+ #include <cstring>
+ #include <cstdio>
+diff --git a/tools/obj_rec_ransac_orr_octree_zprojection.cpp b/tools/obj_rec_ransac_orr_octree_zprojection.cpp
+index 446d53511..42204cfc6 100644
+--- a/tools/obj_rec_ransac_orr_octree_zprojection.cpp
++++ b/tools/obj_rec_ransac_orr_octree_zprojection.cpp
+@@ -62,6 +62,7 @@
+ #include <vtkCubeSource.h>
+ #include <vtkPointData.h>
+ #include <vector>
++#include <chrono>
+ #include <cstdlib>
+ #include <cstring>
+ #include <cstdio>
+diff --git a/tools/obj_rec_ransac_result.cpp b/tools/obj_rec_ransac_result.cpp
+index 884dea61b..78b54eb77 100644
+--- a/tools/obj_rec_ransac_result.cpp
++++ b/tools/obj_rec_ransac_result.cpp
+@@ -57,6 +57,7 @@
+ #include <vtkRenderer.h>
+ #include <vtkRenderWindow.h>
+ #include <vtkTransform.h>
++#include <chrono>
+ #include <cstdio>
+ #include <list>
+ #include <thread>
+diff --git a/tools/obj_rec_ransac_scene_opps.cpp b/tools/obj_rec_ransac_scene_opps.cpp
+index 70ddfe42a..a1d5099a8 100644
+--- a/tools/obj_rec_ransac_scene_opps.cpp
++++ b/tools/obj_rec_ransac_scene_opps.cpp
+@@ -53,6 +53,7 @@
+ #include <vtkDataArray.h>
+ #include <vtkPointData.h>
+ #include <vtkHedgeHog.h>
++#include <chrono>
+ #include <cstdio>
+ #include <thread>
+ 
+diff --git a/visualization/include/pcl/visualization/impl/registration_visualizer.hpp b/visualization/include/pcl/visualization/impl/registration_visualizer.hpp
+index 884735e4a..4a40b11f5 100644
+--- a/visualization/include/pcl/visualization/impl/registration_visualizer.hpp
++++ b/visualization/include/pcl/visualization/impl/registration_visualizer.hpp
+@@ -38,6 +38,7 @@
+ 
+ #pragma once
+ 
++#include <chrono>
+ #include <thread>
+ 
+ 

--- a/ports/pcl/portfile.cmake
+++ b/ports/pcl/portfile.cmake
@@ -11,6 +11,7 @@ vcpkg_from_github(
         install-layout.patch
         install-examples.patch
         fix-clang-cl.patch
+        add-chrono-includes.patch
 )
 
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" PCL_SHARED_LIBS)

--- a/ports/pcl/vcpkg.json
+++ b/ports/pcl/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "pcl",
   "version": "1.15.0",
+  "port-version": 1,
   "description": "Point Cloud Library (PCL) is open source library for 2D/3D image and point cloud processing.",
   "homepage": "https://github.com/PointCloudLibrary/pcl",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6994,7 +6994,7 @@
     },
     "pcl": {
       "baseline": "1.15.0",
-      "port-version": 0
+      "port-version": 1
     },
     "pcre": {
       "baseline": "8.45",

--- a/versions/p-/pcl.json
+++ b/versions/p-/pcl.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "a3842f1300f7fa0b46da6784433678e964520ddf",
+      "version": "1.15.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "62b18d0fdff587cc5f1e0253c109807d47a69723",
       "version": "1.15.0",
       "port-version": 0

